CHANGELOG.md

```diff
@@ -55,3 +56,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.6.0]
 ### Added
 - Support new versions of ethereumjs/tx ([#68](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/68))
+
+[Unreleased]: git+https://github.com/MetaMask/eth-ledger-bridge-keyring/compare/v0.15.0...HEAD
```

Comment:
Hmm, this seems to be malformed. These should be links, without the `git+`

Reply:
I just found out that `@metamask/auto-changelog` wanted those links to be like that because the `repository` section in the `package.json` reported the repo URL to be in that format (git+...). 

I just changed both of the files